src/main/resources/test_files/runtime/FunctionPartialApplication/PartialFunctionApplication-Error2.jq

```diff
@@ -0,0 +1,2 @@
+(:JIQS: ShouldCrash; ErrorCode="SPRKIQ0002";  ErrorMetadata="LINE:2:COLUMN:0:" :)
+max(?)
```

Comment:
that should be an open issue (it's supposed to work in the long term as builtin function behave like user defined functions in this respect)

Reply:
I had already added this to the existing issue of built-in functions not fitting to the functionItem format. https://github.com/RumbleDB/rumble/issues/346 It is listed under Edit1 as I felt these closely relate. Would you like to have this extracted to a separate issue ?

Reply:
Thanks for the feedback -- no need for a separate issue, it's fine that way.